### PR TITLE
Add CS1101S AY19/20 midterm cheatsheet

### PR DESCRIPTION
**Pull request by chrisgzf**
Hello there,

I used your template (and some of your material) to do up my cheatsheet for CS1101S, so I thought I should contribute to your repo, in hope that it might help some people.

My PR includes the LaTeX file as well as the compiled pdf.

Will perhaps submit another PR to add it to the README sometime soon.

Thanks!

### CODE REVIEW

File: src/cs1101s-midterms-2019s1.tex [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+\usepackage{listings}
+
+\begin{document}
+\setlength\parindent{0pt} %TODO this somehow messes up paragraph vertical spacing as well?
+\scriptsize
+% \tiny
+\pagenumbering{gobble}
+
+\begin{center}
+{\large CS1101S Midterms Cheatsheet AY19/20}\\{by chrisgzf (adapted from https://github.com/ning-y/Cheatsheets/)}
```
Comment (ning-y) [inline src/cs1101s-midterms-2019s1.tex]: You can omit the 'adapted from...' if you'd like

**Review by ning-y — APPROVED** [review]
Thanks for contributing! I can merge this in right now if you'd like, or if you want to try your hand at generating the previews and updating the README.md, I can defer it till you're done with that.

**Comment by chrisgzf** [comment]
Updated!

**Review by ning-y — APPROVED** [review]

**Comment by ning-y** [comment]
Thanks!

**Pull request merged**